src/ui/molecules/card/index.jsx

```diff
@@ -15,13 +21,12 @@ export const Card = ({ result, children }) => {
       )}
       <h2>
         <a href={url}>{title}</a>
-        {repo && (
-          <>
-            <span> | </span>
-            <a href={repo}>Repo</a>
-          </>
-        )}
       </h2>
+      {repo && (
+        <a href={repo} className={getHost()}>
```

Comment:
Ah, I see why you used the class style. This should fix the problem.

```suggestion
        <a href={repo} className={getHost().toLowerCase()}>
```